homework/grayscale-image/compression.cpp

```diff
@@ -0,0 +1,45 @@
+#include "compression.hpp"
+
+std::array<std::array<uint8_t, width>, height> decompressGrayscale(std::vector<std::pair<uint8_t, uint8_t>> vec) {
+    std::array<std::array<uint8_t, width>, height> resArr;
+    
+    int num1 = 0;
+    int num2 = 0;
```

Comment:
find better names for this 2 variables :) try to describe what they are doing/ what is their responsibility.